docs/es/features/user_content_translations.md

```diff
@@ -2,59 +2,57 @@
+Este servicio tiene como objetivo poder ofrecer todos los contenidos dinámicos de la aplicación (propuestas, debates, inversiones presupuestarias y comentarios) en diferentes idiomas sin la necesidad de que un usuario o un administrador haya creado cada una de sus traducciones.
 
 Cuando un usuario accede a una pantalla con un idioma donde parte del contenido dinámico que esta visualizando no tiene traducciones, dispondrá de un botón para solicitar la traducción de todo el contenido. Este contenido se enviará a un traductor automático (en este caso [Microsoft TranslatorText](https://azure.microsoft.com/es-es/products/cognitive-services/translator/)) y en cuanto se obtenga la respuesta, todas estas traducciones estarán disponibles para cualquier usuario.
 
-### Como empezar
+### Cómo empezar
 
 Para poder utilizar esta funcionalidad es necesario realizar los siguientes pasos:
 
-1. Disponer de una api key para conectarse con el servicio de traducción. Para ello necesitamos una [cuenta en Azure](https://azure.microsoft.com/es-es/)
-1. Una vez que haya iniciado sesión en el portal de Azure, subscríbase a Traductor en Cognitive Services.
-1. Una vez subscrito al servicio de Translator Text, tendrá accesibles 2 api keys en la sección **Administración de recursos > Claves y punto de conexión** que serán necesarias para la configuración del servicio de traducciones en su aplicación.
+1. Crear una [cuenta en Azure](https://azure.microsoft.com/es-es/)
+2. Una vez que haya iniciado sesión en el portal de Azure, crear un recurso del tipo _Translator Text_
+3. Una vez subscrito al servicio de Translator Text, tendrá accesibles 2 api keys en la sección **Administración de recursos > Claves y punto de conexión** que serán necesarias para la configuración del servicio de traducciones en su aplicación.
 
 ### Configuración
 
-Para activar el servicio de traducciones en su aplicación debe completar los siguientes pasos
+Para activar el servicio de traducciones en su aplicación debe completar los siguientes pasos:
 
 #### Añadir api key en la aplicación
 
-En el apartado anterior hemos comentado que una vez subscritos al servicio de traducciones disponemos de 2 api keys. Para configurar el servicio correctamente en nuestra aplicación deberemos añadir una de las dos api keys en el archivo `secrets.yml` en la sección `apis:` con la key `microsoft_api_key` como podemos ver en la siguiente imágen:
+En el apartado anterior hemos comentado que una vez subscritos al servicio de traducciones disponemos de 2 api keys. Para configurar el servicio correctamente en nuestra aplicación deberemos añadir una de las dos api keys en el archivo `secrets.yml` en la sección `apis:` con la key `microsoft_api_key` como podemos ver en la siguiente imagen:
```

Comment:
For consistency, since we say both "2" and "dos":

```suggestion
En el apartado anterior hemos comentado que una vez subscritos al servicio de traducciones disponemos de dos _API keys_. Para configurar el servicio correctamente en nuestra aplicación deberemos añadir una de las dos  _API keys_ en el archivo `secrets.yml`, en la sección `apis:`, con la clave `microsoft_api_key`, como podemos ver en la siguiente imagen:
```